### PR TITLE
Added list of missing features to WinUI 2 article

### PR DESCRIPTION
**Pull request by jasonstephen15**
**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2?branch=pr-en-us-1609

Writer/Editor pass: vchapel

### CODE REVIEW

**Comment by vchapel** [comment]
@jasonstephen15 you can see the link suggestion. I'm not sure if you need it to be a specific version or not. Other than that I think it looks good.

File: microsoft-edge/webview2/get-started/winui2.md [hunk 1]
```diff
@@ -11,10 +11,23 @@ keywords: WebView2, webview2, WebView, webview, winui apps, winui, edge, CoreWeb
...
 
 > [!NOTE]
-> The WinUI 2 package depends on a prerelease WebView2 package. For full API compatibility, use a preview browser channel as your runtime such as the Beta, Dev, or Canary channel of Microsoft Edge.
+> The [WinUI 2 package](https://www.nuget.org/packages/Microsoft.UI.Xaml/2.8.0-prerelease.210927001) takes a dependency on a prerelease WebView2 package. For full API compatibility, use a preview browser channel as your runtime such as the Beta, Dev, or Canary channel of Microsoft Edge.
+
+> [!Important]
+> The WebView2 WinUI Control is actively being developed. There are some features that will be missing or broken. Some of these include:
+> * Download UI -- Note: in order to maintain compatability, developers should manually disable this feature by intercepting the [download starting event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2downloadstartingeventargs?view=webview2-winrt-1.0.1054.31).
```
Comment (vchapel) [inline microsoft-edge/webview2/get-started/winui2.md:21]:
```suggestion
> * Download UI -- To maintain compatibility, developers should manually disable this feature by intercepting the [download starting event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2downloadstartingeventargs?view=webview2-winrt-1.0.1054.31).
```

File: microsoft-edge/webview2/get-started/winui2.md [hunk 2]
```diff
@@ -11,10 +11,23 @@ keywords: WebView2, webview2, WebView, webview, winui apps, winui, edge, CoreWeb
...
+In this article, you will get started creating your first WebView2 app in WinUI 2, and learn about the main features of WebView2. For more information about individual WebView2 APIs, navigate to [Microsoft Edge WebView2 API Reference](../webview2-api-reference.md) and then select the WinRT reference links.
 
 > [!NOTE]
-> The WinUI 2 package depends on a prerelease WebView2 package. For full API compatibility, use a preview browser channel as your runtime such as the Beta, Dev, or Canary channel of Microsoft Edge.
+> The [WinUI 2 package](https://www.nuget.org/packages/Microsoft.UI.Xaml/2.8.0-prerelease.210927001) takes a dependency on a prerelease WebView2 package. For full API compatibility, use a preview browser channel as your runtime such as the Beta, Dev, or Canary channel of Microsoft Edge.
+
+> [!Important]
+> The WebView2 WinUI Control is actively being developed. There are some features that will be missing or broken. Some of these include:
```
Comment (champnic) [inline microsoft-edge/webview2/get-started/winui2.md:20]: "WinUI 2"

File: microsoft-edge/webview2/get-started/winui2.md [hunk 3]
```diff
@@ -11,10 +11,23 @@ keywords: WebView2, webview2, WebView, webview, winui apps, winui, edge, CoreWeb
...
+> * Download UI -- To maintain compatibility, developers should manually disable this feature by intercepting the [download starting event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2downloadstartingeventargs?view=webview2-winrt-1.0.1054.31).
```
Comment (champnic) [inline microsoft-edge/webview2/get-started/winui2.md:21]: I think we should expand on why there's a potential "compatibility" issue here.

File: microsoft-edge/webview2/get-started/winui2.md [hunk 4]
```diff
@@ -11,10 +11,23 @@ keywords: WebView2, webview2, WebView, webview, winui apps, winui, edge, CoreWeb
...
+> * Autofill UI
+> * File Picker Dialog
+> * Save Screenshot
```
Comment (champnic) [inline microsoft-edge/webview2/get-started/winui2.md:24]: I don't think this is an issue is it? It's the host app side of things that needs to make sure they are using UWP functions for saving a file, which they should be doing anyways.

**Review by champnic — CHANGES_REQUESTED** [review]

**Comment by jasonstephen15** [comment]
@vchapel - this is good to merge

**Review by vchapel — APPROVED** [review]
LGTM

**Pull request merged**